packages/itwinui-css/src/dialog/dialog.scss

```diff
@@ -59,7 +59,7 @@ $iui-dialog-min-height: 7.75rem; // 7.75rem = 124px = title bar height + margins
 
 .iui-dialog-default {
   max-width: Max(50%, $iui-dialog-width);
-  min-width: Min(100%, $iui-dialog-width);
```

Comment:
this `Min` function was added intentionally, because we don't want the dialog to overflow the container or the screen (mobile phones can be narrower than 380px).